Lombiq.Hosting.Tenants.Maintenance.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -22,4 +23,14 @@ public static async Task TestSiteOwnerPermissionToRoleMaintenanceExecutionAsync(
         await context.GoToAdminRelativeUrlAsync("/Roles/Edit/Editor");
         context.Get(By.Id("Checkbox.SiteOwner")).GetDomProperty("checked").ShouldBe(bool.TrueString);
     }
+
+    public static async Task ChangeUserSensitiveContentMaintenanceExecutionAsync(this UITestContext context)
+    {
+        var loginPage = await context.GoToLoginPageAsync();
+        (await loginPage.LogInWithAsync(context, DefaultUser.UserName, DefaultUser.Password))
+            .ShouldStayOnLoginPage()
+            .ValidationSummaryErrors.Should.Not.BeEmpty();
+
+        (await context.GetCurrentUserNameAsync()).ShouldBeEmpty();
+    }
```

Comment:
Instead of checking this, we should test a @lombiq email address (test if it doesn't changes), and then you can login with it and check directly if the original `admin` user's name changed.